internal/cli/atlas/alerts/settings/list_test.go

```diff
@@ -61,9 +61,8 @@ func TestConfigList_Run(t *testing.T) {
 	}
 
 	params := &admin.ListAlertConfigurationsApiParams{
-		GroupId:      listOpts.ProjectID,
-		ItemsPerPage: &listOpts.ItemsPerPage,
```

Comment:
Why?

Reply:
Is that breaking change?

Reply:
This is to avoid sending `itemsPerPage = 0`

Reply:
It looks like bug in the code. We need to connect ItemsPerPage with CLI default value